src/HealthGPS/energy_balance_model.cpp

```diff
@@ -78,131 +82,169 @@ void EnergyBalanceModel::update_risk_factors(RuntimeContext &context) {
             continue;
         }
 
-        simulate_person(person, true, final_shift);
+        // TODO: Compute and update risk factors.
+        simulate_person(person, shift);
+        // SimulatePersonState state = simulate_person(person, shift);
+        // person.risk_factors[H_key] = state.H;
+        // person.risk_factors[BW_key] = state.BW;
+        // person.risk_factors[PAL_key] = state.PAL;
+        // person.risk_factors[RMR_key] = state.RMR;
+        // person.risk_factors[F_key] = state.F;
+        // person.risk_factors[L_key] = state.L;
+        // person.risk_factors[ECF_key] = state.ECF;
+        // person.risk_factors[G_key] = state.G;
+        // person.risk_factors[W_key] = state.W;
+        // person.risk_factors[EE_key] = state.EE;
+        // person.risk_factors[EI_key] = state.EI;
```

Comment:
I'm guessing you left this as a placeholder? I think it would be better to have a dedicated function for updating the risk factors from your `state` struct. It *could* potentially be a member method of `Person`.